--- Makefile.in	2018-07-03 04:16:25.000000000 -0700
+++ Makefile.in.new	2019-05-01 17:06:39.030127918 -0700
@@ -114,11 +114,6 @@
 check-root:
 	@test `id -u` = '0' || (echo "You must run this command as root"; exit 1)
 
-check-ldconfig:
-	@test "x/usr/@ARCHLIB@" = "x$(libdir)" || \
-	    grep -l -R "^$(libdir)$$" /etc/ld.so.conf* >/dev/null 2>&1 || \
-	    echo "Add manually $(libdir) to /etc/ld.so.conf and run ldconfig (as root)"
-
 clean-config-files: check-topdir
 	-rm -rf auto*.cache
 	-rm -f @CONFIGURE_FILES@
@@ -134,7 +129,7 @@
 cvsclean: check-topdir clean clean-apidocs clean-packing clean-config-files
 	-rm -f configure yate-config.in
 
-.PHONY: engine libs ilibs modules clients test apidocs-build apidocs-kdoc apidocs-doxygen apidocs-everything check-topdir check-ldconfig windows
+.PHONY: engine libs ilibs modules clients test apidocs-build apidocs-kdoc apidocs-doxygen apidocs-everything check-topdir windows
 engine: library libyate.so $(PROGS)
 
 apidocs-kdoc: check-topdir
@@ -205,7 +200,7 @@
 	@cmp -s yateiss.inc $@/yateiss.inc || cp -p yateiss.inc $@/yateiss.inc
 
 .PHONY: install install-root install-noconf install-noapi install-api uninstall uninstall-root
-install install-root: all $(APIDOCS) install-noapi install-api check-ldconfig
+install install-root: all $(APIDOCS) install-noapi install-api
 
 install-noapi: install-noconf
 	$(MAKE) -C ./conf.d install
