--- engine/Mutex.cpp	2018-07-03 04:16:38.000000000 -0700
+++ engine/Mutex.cpp.new	2019-05-01 15:28:46.578800837 -0700
@@ -32,12 +32,8 @@
 
 #ifdef MUTEX_HACK
 extern "C" {
-#if defined(__FreeBSD__) || defined(__OpenBSD__) || defined(__NetBSD__) || defined(__APPLE__)
 extern int pthread_mutexattr_settype(pthread_mutexattr_t *__attr, int __kind);
 #define PTHREAD_MUTEX_RECURSIVE_NP PTHREAD_MUTEX_RECURSIVE
-#else
-extern int pthread_mutexattr_settype(pthread_mutexattr_t *__attr, int __kind) __THROW;
-#endif
 }
 #endif
 
