--- yateclass.h	2018-07-03 04:16:42.000000000 -0700
+++ yateclass.h.new	2019-05-01 16:55:12.850679109 -0700
@@ -2928,20 +2928,6 @@
 YATE_API String operator+(const char* s1, const String& s2);
 
 /**
- * Prevent careless programmers from overwriting the string
- * @see TelEngine::String::operator=
- */
-inline const char *strcpy(String& dest, const char* src)
-    { dest = src; return dest.c_str(); }
-
-/**
- * Prevent careless programmers from overwriting the string
- * @see TelEngine::String::operator+=
- */
-inline const char *strcat(String& dest, const char* src)
-    { dest += src; return dest.c_str(); }
-
-/**
  * Utility function to look up a string in a token table,
  * interpret as number if it fails
  * @param str String to look up

